src/agGrid/agGridInteractions.js

```diff
@@ -347,14 +360,16 @@ function _filterBySearchTextColumnFloatingFilter(agGridElement, options) {
  * @param options.searchCriteria.columnName [REQUIRED] name of the column to filter
  * @param options.searchCriteria.filterValue [REQUIRED] value to input into the filter textbox
  * @param options.hasApplyButton [Optional] True if "Apply" button is used, false if filters by text input automatically.
+ * @param options.noMenuTabs [Optional] True if you use for example the community edition of ag-grid, which has no menu tabs
  */
 export function filterByCheckboxColumnMenu(agGridElement, options) {
   // Check if there are multiple search criteria provided by attempting to access the columnName
   if (!options.searchCriteria.columnName) {
     options.searchCriteria.forEach((_searchCriteria) => {
       const _options = populateSearchCriteria(
         _searchCriteria,
-        options.hasApplyButton
+        options.hasApplyButton,
+        otions.noMenuTabs
```

Comment:
This guy appears to be a typo and is failing one of the tests.
```suggestion
        options.noMenuTabs
```